kolibri_explore_plugin/assets/src/views/SearchPage.vue

```diff
@@ -0,0 +1,167 @@
+      <b-container v-if="recommended" class="pb-5 pt-3">
+        <h4 class="text-muted">
+          Featured Channels
+        </h4>
+        <ChannelCardGroup
+          :rows="recommended"
+          :hasThumbnail="true"
+          :columns="3"
+          @card-click="goToChannel"
+        />
+      </b-container>
+    </div>
+  </div>
+
+</template>
+
+
+<script>
+
+  import _ from 'underscore';
+
+  import { mapMutations, mapState } from 'vuex';
+
+  import { Header, ChannelCardGroup, SearchBar } from 'eos-components';
+  import { PageNames } from '../constants';
+
+  import { searchChannels } from '../modules/topicsRoot/handlers';
+
+  export default {
+    name: 'SearchPage',
+    components: {
+      ChannelCardGroup,
+      Header,
+      SearchBar,
+    },
+    data() {
+      return {
+        query: '',
+      };
+    },
+    computed: {
+      ...mapState('topicsRoot', { searchResult: 'searchResult', channels: 'rootNodes' }),
+      ...mapState({
+        loading: state => state.core.loading,
+      }),
+      recommended() {
+        if (!this.channels || !this.channels.length) {
+          return null;
+        }
+
+        const channels = [];
+        const allChannels = [...this.channels];
+        while (channels.length < 3 && allChannels.length > 0) {
+          const index = _.random(0, allChannels.length - 1);
+          const [channel] = allChannels.splice(index, 1);
+          channels.push(channel);
+        }
+        return [channels];
+      },
+      cleanedQuery() {
+        return this.query.trim();
+      },
+      hasResults() {
+        return this.searchResult.channels || this.searchResult.result;
+      },
+      resultChannels() {
+        const { channels } = this.searchResult;
+        if (!channels || !channels.length) {
+          return null;
+        }
+
+        return _.chunk(this.searchResult.channels, 3);
+      },
+    },
+    watch: {
+      cleanedQuery() {
+        this.search(this.cleanedQuery);
+      },
+    },
+    mounted() {
+      this.setSearchResult({});
+    },
+    methods: {
+      ...mapMutations({
+        setSearchResult: 'topicsRoot/SET_SEARCH_RESULT',
+      }),
+      goBack() {
+        this.$router.push({
+          name: PageNames.TOPICS_ROOT,
+        });
+      },
+      goToChannel(channelId) {
+        this.$router.push({
+          name: PageNames.TOPICS_CHANNEL,
+          params: { channel_id: channelId },
+        });
+      },
+      search(query) {
+        if (!query) {
+          this.setSearchResult({});
+          return;
+        }
+
+        // TODO: show loading
+        searchChannels(this.$store, { search: query });
+      },
+    },
+  };
+
+</script>
+
+
+<style lang="scss" scoped>
+
+  @import '../styles';
+
+  .main {
+    background-color: white;
+  }
+
+  .header {
+    background: white;
+  }
+
+  .search-page {
+    padding-top: $navbar-height;
+  }
+
+  .recommended {
+    background-color: $gray-300;
```

Comment:
Oh I used gray-200 for the theme body. I need to check Figma again. In any case, this should be `$body-bg`.